base/mpfr.jl

```diff
@@ -151,6 +151,11 @@ global precision; `convert` will always return `x`.
 convenience since decimal literals are converted to `Float64` when parsed, so
 `BigFloat(2.1)` may not yield what you expect.
 
+See also:
```

Comment:
Just want to clarify: is the proposed style to use the colon at the start of a multi-line list, but not otherwise?

Reply:
Yes, in this case, it felt more grammatical with the colon because it is a multiline list.